apps/web/src/config/__tests__/vaults.test.ts

```diff
@@ -0,0 +1,72 @@
+import { ChainId } from '@pancakeswap/chains'
+import { vaultsConfigChainMap } from '@pancakeswap/position-managers/constants'
+import { positionManagerAdapterABI, positionManagerWrapperABI } from '@pancakeswap/position-managers/src/abi'
+import { pancakeV3PoolABI } from '@pancakeswap/v3-sdk'
+import { publicClient } from 'utils/client'
+import { describe, expect, it } from 'vitest'
+
+const mainnetVaults = [vaultsConfigChainMap[ChainId.BSC], vaultsConfigChainMap[ChainId.ETHEREUM]]
+
+function hasDuplicates(array: any[]) {
+  return new Set(array).size !== array.length
+}
+
+describe('Config position manger Vault', () => {
+  it.each(mainnetVaults)('All farm has an unique pid', (...vaults) => {
```

Comment:
Should be unique `id` right?  And this is `position manager` not `farm`

Reply:
yes, let me update it